packages/dx-react-grid-bootstrap4/src/templates/table-tree-cell.test.jsx

```diff
@@ -33,4 +33,18 @@ describe('TableTreeCell', () => {
     expect(tree.find('.d-flex.flex-direction-row.align-items-center').exists())
       .toBeTruthy();
   });
+
+  it('should apply align classes', () => {
+    let tree = shallow((
+      <TableTreeCell tableColumn={{ align: 'right' }} />
+    ));
+    expect(tree.find('.d-flex.flex-direction-row.align-items-center.text-right').exists())
+      .toBeTruthy();
+
+    tree = shallow((
```

Comment:
It's better to split it to 2 tests

Reply:
Why? Can you describe your position?

I think that it's one feature and it should have one test.

Reply:
It's inconsistent with common test pattern: arrange -> act -> assert. In our case we should stick to single shallow/mount per test